package.json

```diff
@@ -61,13 +53,10 @@
     "lint": "eslint --ext .js,.jsx src",
     "build": "npm run build:js && npm run build:scss",
     "build:js": "webpack -d",
-    "postbuild:js": "uglifyjs build/js/lyra.js -cm > build/js/lyra.min.js",
     "build:scss": "node-sass --include-path src/scss src/scss/main.scss build/css/main.css",
-    "postbuild:scss": "cleancss -o build/css/main.min.css build/css/main.css",
```

Comment:
I've created #267 to track adding this minification back in; our dev server and build process don't currently account for the minified files so these steps are currently just baggage, hence removing them rather than simply leaving them be.